tests/integration_tests/functional/test_drive_vhost_user.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
the integration test and seccomp update should be in separate commits

Reply:
done